src/OrleansServiceBus/Providers/Streams/EventHub/EventHubAdapterFactory.cs

```diff
@@ -283,6 +283,12 @@ private EventHubAdapterReceiver GetOrCreateReceiver(QueueId queueId)
             return cache;
         }
 
+        protected virtual IEventHubQueueCache CreateCache(IStreamQueueCheckpointer<string> checkpointer, 
```

Comment:
@xiazen I don't have a problem extending `EventHubQueueCache`, but I don't want to duplicate logic creating `FixedSizeObjectPool<FixedSizeBuffer>`, `TimePurgePredicate` and then the whole `CreateCacheFactory` method